tests/unit/commands/test_initialize.py

```diff
@@ -0,0 +1,107 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
```

Comment:
same question here: are there specific changes that I should check here?

Reply:
It's a rewrite from `unit.TestCase` to `pytest`, but there's not one thing that stands out but everything stands out :laughing: